DoLahTest/ParserUnitTest.cpp

```diff
@@ -379,6 +387,130 @@ namespace DoLahTest {
         }
     }
 
+    // From here, integrated test on special cases of 'toDateFormat' for relative time descriptions.
+    // The time will auto-filled, and the behaviour changes depending on the time of running.
+    // Test will be disabled in a certain condition where it cannot run properly.
+    TEST_METHOD(DateGivenDayAndMonth) {
+        if (month == 1 && day == 1) {
+            Assert::IsTrue(true);
```

Comment:
Are you able to mock the current time? because relying on time is a bad things in testing. (it pass/fail, but not guaranteed accurate)